internal/olm/operator/registry/operator_installer.go

```diff
@@ -144,7 +144,7 @@ func (o OperatorInstaller) UpgradeOperator(ctx context.Context) (*v1alpha1.Clust
 	log.Infof("Found existing catalog source with name %s and namespace %s", cs.Name, cs.Namespace)
 
 	// Update catalog source
-	err := o.CatalogUpdater.UpdateCatalog(ctx, cs, subscription)
+	err := o.CatalogUpdater.UpdateCatalog(ctx, cs, subscription, skipTLSVerify, useHTTP)
```

Comment:
```suggestion
	err := o.CatalogUpdater.UpdateCatalog(ctx, cs, subscription)
```
The IndexImageCatalogCreator has the fields on it already. You can use those.